docs/why-polar-signals.md

```diff
@@ -0,0 +1,72 @@
+
+Using the compare mode we can see down to the line number where the difference in CPU time is. Comparing can be done with any two queries, it can be the same process at two different points in time, but it can also be the combined CPU profiling data of version X compared to the combined profiling data of version Y.
+
+Most companies have never seen profiling data of their infrastructure and immediately see 20-30% low hanging fruit that are easy to get rid of.
+
+### Reduce Cost
+
+We recommend deploying the agent on every host, as that allows all CPU profiling data to be collected for the entire infrastructure. Looking at all profiling data of all processes combined, we can easily spot which code paths are even worth optimizing in order to reduce cost. Without system-wide profiling, you cannot tell what is even worth optimizing in the grand scheme of things.
+
+## Key Features
+
+There are a couple of key features that make the above use cases possible and Polar Signals Cloud unique as a product.
+
+### Zero Instrumentation & Wide Language Support
+
+Getting started with Polar Signals Cloud is incredibly easy, thanks to our zero instrumentation profiler. No libraries needed, the agent automatically start profiling every process, regardless of the language.
+
+We've put incredible amounts of work to build this state-of-the-art profiler using [novel techniques](https://www.polarsignals.com/blog/posts/2022/11/29/profiling-without-frame-pointers/) that make it faster and more secure than previously known profilers!
+
+### Low Overhead eBPF Profiler
+
+Even in heavy CPU stress tests overhead does not exceed 1%, in most deployments overhead is indistinguishable from noise. This is made possible thanks to implementing our profiler using eBPF.
+
+### Open Source
+
+All code that runs in customer environments is 100% open source under the Apache 2 license. See the [Parca Agent](https://github.com/parca-dev/parca-agent) open source project.
+
+### Prometheus-like Query Language
+
+Slice-and-dice continuous profiling data using the selector-based language you are already know and love from Prometheus. Add arbitrary dimensions to your data to make it reflect your organization, the tool will not force your organization to change.
+
+### Global View
+
+Polar Signals Cloud allows you to view all profiling data across your entire infrastructure, this enabled powerful workflows that allow you to find things to optimize that actually matter and will make a difference.
+
+### Filter by function
+
+You already know what you want to optimize? Great! Use the filter-by-function functionality
+
+### Compare
+
+Polar Signals Cloud allows comparing the result of any two queries against each other. Compare the CPU time of version X with version Y (to understand a performance regression), compare datacenter A with datacenter B, or just understand what code was executing at one point in time of a process vs. another point in time.
+
+## Polar Signals Cloud vs. Parca
```

Comment:
I would maybe roll this up into the top why polar signals cloud section. Get it out of the way first that this is a hosted version of Parca and it scales with your business without having to maintain the software etc.

Reply:
Sounds good!